### DISCUSSION TITLE
Alpha Blend mode for materials

### DISCUSSION THREAD

**Discussion by Georgianamarin05** [frzi/Model3DView#2] [Q&A] [answered]
Hi,

I am trying to generate an avatar and I have an issue. The material for hair area is not transparent when i am rendering it with Model3DView. Do you have any idea how can I fix it?

**Comment by frzi** [comment, marked as answer]
Hmm if this is concerning a glTF model then this is probably something for [GLTFSceneKit](https://github.com/magicien/GLTFSceneKit), the package that converts the glTF to a SceneKit asset.

If you’re working directly with a SceneKit asset then you may find your answer in the SceneKit community. As far as I’m aware it’s a material feature.
